### PR TITLE
Draft: Add config file for the Apicurio bot

### PR DESCRIPTION
**Pull request by jsenko**
@EricWittmann @carlesarnal Take a look please, and add what you think may be missing (at least from the start), so we can start using the bot. Let me know what needs to improve, I've already added a few TODOs and I'll update the bot before we merge this.

### CODE REVIEW

File: .github/apicurio-bot.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        - EricWittmann
+
+    # SerDes
+    - labels:
+        - area/serdes
+      patterns:
+        title:
+          - 'serdes'
+        files:
+          - '^serdes'
```
Comment (carlesarnal) [inline .github/apicurio-bot.yaml:82]: Feel free to notify me here. I'm probably familiar enough to answer most questions around serdes now.

**Comment by carlesarnal** [comment]
I think this is a good start. We can merge this and evolve it over time.

**Comment by jsenko** [comment]
Cool, I want to implement these improvements, and then I'll merge this:
- Improve label management, so we can better specify which labels are e.g. area labels
- Option to explicitly skip notifying anyone
- Ignoring case in labels

**Comment by andreaTP** [comment]
🎉

**Pull request merged**